lib/nolibc/include/netdb.h

```diff
@@ -0,0 +1,14 @@
+#ifndef _NETDB_H_
+#define _NETDB_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+extern int h_errno;
```

Comment:
Usually this is a macro for `*__h_errno_location()` that returns the correct value and that function is also defined in the other file. But it's not used for some reason. 
Did you forget to do it that way or is this intentional?